sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -64,6 +67,12 @@ export class StudentController extends BaseController {
         provinceState: existingStudent.contactInfo.addresses[0].province,
         phone: existingStudent.contactInfo.phone,
       },
+      sin: existingStudent.sin,
+      pdVerified: existingStudent.studentPDVerified,
+      validSin: existingStudent.validSIN ?? false,
```

Comment:
When validSIN is null it means that it was not validated yet, we should not assume that it is false.
false indicates that it was validated and is not valid.

Reply:
oh..okay. Thanks for that (Y)